enso.cfg

```diff
@@ -0,0 +1,14 @@
+[#]
+sets = ["enso_diags"]
+case_id = "Regression on Nino3.4"
+
+contour_levels= [-8, -6, -4, -2, 0, 2, 4]
```

Comment:
Are these the contours we'll always want? Is there a way to auto-generate contours such that they are the same for both `test` and `ref`?

Reply:
The contours should be variable depended...the values for `test` and `ref` are always same, using the parameter `contour_levels`